google/web-stories-wp
# Saved Templates: Only show scrollbar when there are many templates

## Context



## Summary

Remove scrollbar until we need it on the saved templates.

## Relevant Technical Choices

n/a

## To-do

n/a

## User-facing changes

<img src="https://user-images.githubusercontent.com/22185279/138316256-b8706b70-7339-4d23-b13d-f400d78f4a27.gif" height="400px" />

## Testing Instructions






This PR can be tested by following these steps:

1. Create a new story
2. Add something to a page
3. Go to page templates tab in the library
4. Switch to `Saved Templates`
5. Save the current page as a template
6. See that there's no scrollbar
7. Keep saving until a scrollbar appears

## Reviews

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist




---



Fixes #9399
